bosh/alerts/awslogs.alerts

```diff
@@ -0,0 +1,32 @@
+ALERT AWSLogsGroupExpired
```

Comment:
WDYT about emitting a timestamp of the last successful log group check and alerting if too much time has passed?

Reply:
✔️